Solutions/Azure Active Directory/Analytic Rules/BruteForceCloudPC.yaml

```diff
@@ -27,21 +27,20 @@ query: |
   | where AppDisplayName =~ "Windows Sign In"
   // Split out failure versus non-failure types
   | extend FailureOrSuccess = iff(ResultType in ("0", "50125", "50140", "70043", "70044"), "Success", "Failure")
```

Comment:
We could probably vastly simplify this with use of imAuthentication and series_decompose_anomalies

Reply:
Yeah I think so - make a series of the diff.

Reply:
@petebryan Updated the query with series_decompose_anomalies